backend/LexBoxApi/ScheduledTasksKernel.cs

```diff
@@ -0,0 +1,64 @@
+﻿using AppAny.Quartz.EntityFrameworkCore.Migrations;
+using AppAny.Quartz.EntityFrameworkCore.Migrations.PostgreSQL;
+using CrystalQuartz.Application;
+using CrystalQuartz.Application.Startup;
+using CrystalQuartz.AspNetCore;
+using LexBoxApi.Jobs;
+using LexData;
+using LexData.Configuration;
+using Quartz;
+using Quartz.AspNetCore;
+using Quartz.Impl.AdoJobStore;
+
+namespace LexBoxApi;
+
+public static class ScheduledTasksKernel
+{
+    public static void AddScheduledTasks(this IServiceCollection services, IConfiguration configuration)
+    {
+        services.ConfigureDbModel(builder =>
+        {
+            builder.AddQuartz(b => b.UsePostgreSql());
+        });
+        services.AddQuartz(q =>
+        {
+            q.UsePersistentStore(options =>
+            {
+                options.UseProperties = true;
+                options.UseNewtonsoftJsonSerializer();
```

Comment:
It's too bad they don't have a serializer option for using System.Text.Json yet, as I suspect (but would have a hard time proving) that it would be faster than the Newtonsoft JSON serializer. Something to consider for later once/if Quartz gets around to offering the option.

Reply:
Yeah, I looked into providing my own, but didn't want to support it. They do have it coming in their next major release it looks like.